types/react.d.ts

```
@@ -0,0 +1,30 @@
+/// <reference types="react" />
+/// <reference types="react-dom" />
+
+declare module "react" {
+  export * from "@types/react";
+  
+  // Re-export key types for explicit access
+  export type FC<P = {}> = React.FC<P>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)